File: Blueprism/BluePrism.Api/BluePrism.Api.Domain/StageTypes.cs
namespace BluePrism.Api.Domain
{
    using System;

    [Flags]
    public enum StageTypes
    {
        Undefined = 1 << 0,
        Action = 1 << 1,
        Decision = 1 << 2,
        Calculation = 1 << 3,
        Data = 1 << 4,
        Collection = 1 << 5,
        Process = 1 << 6,
        SubSheet = 1 << 7,
        ProcessInfo = 1 << 8,
        SubSheetInfo = 1 << 9,
        Start = 1 << 10,
        End = 1 << 11,
        Anchor = 1 << 12,
        Note = 1 << 13,
        LoopStart = 1 << 14,
        LoopEnd = 1 << 15,
        Read = 1 << 16,
        Write = 1 << 17,
        Navigate = 1 << 18,
        Code = 1 << 19,
        ChoiceStart = 1 << 20,
        ChoiceEnd = 1 << 21,
        WaitStart = 1 << 22,
        WaitEnd = 1 << 23,
        Alert = 1 << 24,
        Exception = 1 << 25,
        Recover = 1 << 26,
        Resume = 1 << 27,
        Block = 1 << 28,
        MultipleCalculation = 1 << 29,
        Skill = 1 << 30,
    }
}

File: TEST data for large repo/REPO_Copy-6/Blueprism/BluePrism.AutomateProcessCore/StageTypes.vb
''' <summary>
''' Enumeration of the different types of stage
''' </summary>
Public Enum StageTypes As Integer
    Undefined = 1
    Action = 2
    Decision = 4
    Calculation = 8
    Data = 16
    Collection = 32
    Process = 64
    SubSheet = 128
    ProcessInfo = 256
    SubSheetInfo = 512
    Start = 1024
    [End] = 2048
    Anchor = 4096
    Note = 8192
    LoopStart = 16384
    LoopEnd = 32768
    Read = 65536
    Write = 131072
    Navigate = 262144
    Code = 524288
    ChoiceStart = 1048576
    ChoiceEnd = 2097152
    WaitStart = 4194304
    WaitEnd = 8388608
    Alert = 16777216
    Exception = 33554432
    Recover = 67108864
    [Resume] = 134217728
    Block = 268435456
    MultipleCalculation = 536870912
    Skill = 1073741824
End Enum
